### PR TITLE
DOC-4067 added FAQ and note about harmless OOM message

### PR DESCRIPTION
**Pull request by andy-stark-redis**
I thought the FAQ was the best place for this and there is also a note in the ingest lifecycle section.

### CODE REVIEW

File: content/integrate/redis-data-integration/ingest/architecture.md [hunk 1]
```diff
@@ -68,6 +68,23 @@ RDI automatically enters a second phase called *change streaming*, where
 changes in the data are captured as they happen. Changes are usually
 added to the target within a few seconds after capture.
 
+## Backpressure mechanism
+
+Sometimes, data records can get added to the streams faster than RDI can
+process them. This can happen if the target is slowed or disconnected
+or simply if the source generates a lot of change data quickly.
+If this continues then the streams will eventually occupy all the
```
Comment (dwdougherty) [inline content/integrate/redis-data-integration/ingest/architecture.md:76]:
```suggestion
If this continues, then the streams will eventually occupy all the
```

File: content/integrate/redis-data-integration/ingest/architecture.md [hunk 2]
```diff
@@ -68,6 +68,23 @@ RDI automatically enters a second phase called *change streaming*, where
 changes in the data are captured as they happen. Changes are usually
 added to the target within a few seconds after capture.
 
+## Backpressure mechanism
+
+Sometimes, data records can get added to the streams faster than RDI can
+process them. This can happen if the target is slowed or disconnected
+or simply if the source generates a lot of change data quickly.
```
Comment (dwdougherty) [inline content/integrate/redis-data-integration/ingest/architecture.md:75]: Just a suggestion...

```suggestion
or simply if the source quickly generates a lot of change data.
```

**Review by dwdougherty — APPROVED** [review]
Apart from a couple of niggles; see below.

**Comment by andy-stark-redis** [comment]
@dwdougherty Cheers!

**Pull request merged**